config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java

```diff
@@ -881,4 +864,15 @@ public MbusParams(Double maxConcurrentFactor, Double documentExpansionFactor, In
         }
     }
 
+    public void setIdentity(Identity identity) {
+        this.identity = identity;
+        addComponent(identity);
+    }
+
+    @Override
+    public void getConfig(IdentityConfig.Builder builder) {
```

Comment:
Why this routing of config? Just to allow your unit test to pass? Switch the test to use the real configId instead!